runtime/vm/jvminit.c

```diff
@@ -6349,6 +6349,40 @@ protectedInitializeJavaVM(J9PortLibrary* portLibrary, void * userData)
 	Xj9BreakPoint("jvminit");
 #endif
 
+        {
+		IDATA enabled = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXPRINTFLAGSFINALENABLE, NULL);
+		IDATA disabled = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXPRINTFLAGSFINALDISABLE, NULL);
+		if (enabled > disabled) {
+			UDATA maxHeapSize, maxDirectMemorySize;
+			char *howset;
```

Comment:
The types of `maxHeapSize` and `maxDirectMemorySize` should be `uint64_t` to match the comment below.
`howset` should have type `const char *`.

Reply:
I'll fix the decls.
> The types of maxHeapSize and maxDirectMemorySize should be uint64_t to match the comment below.
I'm confused by this comment, because  `j9gc_get_maximum_heap_size()` returns UDATA and `vm->directByteBufferMemoryMax` is declared as UDATA.

Reply:
See line 6364 below. Rather than both `uint64_t`, we should have `size_t maxHeapSize;`,

Reply:
Is there a reason I should use `size_t` explicitly rather than just `UDATA`? For most purposes in this PR, size_t is just a string that's printed in the output because that's how the Hotspot output refers to a machine word sized unsigned integer. The values in OpenJ9 are explicitly defined as `UDATA` so I thought to continue to refer to them as such.

Reply:
If you're trying to match the output of an implementation that says `size_t` and `uint64_t` why would you use other types for those locals? The compiler will happily widen the values where necessary. Once you've settled on types for the locals, the choice of formatting must match.

Reply:
I was thinking it felt a strange precedent to have Hotspot's implementation choices determine what types we should use for these values in OpenJ9 but, from a consumer standpoint, having all implementations produce as similar as possible output seems desirable.  Anywhere it will be different, we need to ask ourselves if it really needs to be different, and I don't think it needs to be different in these specific cases. We probably won't get the reverse consideration, but such is life I guess.

Given all that, I'll put them back to `size_t` and `uint64_t`.